src/deploy/osp_deployer/profiles/xsp.json

```diff
@@ -38,6 +38,13 @@
                    "false"
                 ]
              },
+
+        "smart_nic": {
+                "valid_values": [
+                   "true",
```

Comment:
Since we list only false as a valid value for sriov_enable, we should probably list only false as a valid value for smart_nic in the xsp profile

Reply:
Okay will change it.